src/main/java/ttt/board/BoardFactory.java

```diff
@@ -0,0 +1,7 @@
+package ttt.board;
+
+public class BoardFactory {
```

Comment:
This factory is just a thin wrapper around a constructor. Is there any value to it?